### PR TITLE
19000 metrics and colours

### LINKED ISSUE #19000: Enhance metrics page to also show warnings
Page: Overview / "View metrics and history"

Currently all of the graphs at the top of the page only have two color states:  blue (normal), and red (problem).  It would be helpful if there was a warning color between normal and red.  

For example, show warning for filesystems over 80%, and the normal red/problem for filesystems over 90%:

Screenshot from 2023-06-22 15-43-10

### PR DESCRIPTION
**Pull request by ianbrown78**
Fixes #19000 - Make CPU, RAM & Disk utilisation progress bars show as warning when >= 80%.
The screenshot shows the CPU on my machine whilst transcoding video.
Screenshot from 2023-08-22 20-23-39

### CODE REVIEW

**Comment by jelly** [comment]
Alright, all testing farm failures are related to the new tuned rc release.

File: pkg/metrics/metrics.jsx [hunk 1]
```diff
@@ -725,7 +725,7 @@ class CurrentMetrics extends React.Component {
                         value={this.state.swapUsed}
                         className="pf-m-sm"
                         min={0} max={swapTotal}
-                        variant={swapUsedFraction > 0.9 ? ProgressVariant.danger : null}
+                        variant={swapUsedFraction > 0.9 ? ProgressVariant.danger : swapUsedFraction >= 0.8 ? ProgressVariant.warning : null}
```
Comment (cockpituous) [inline pkg/metrics/metrics.jsx:728]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/metrics/metrics.jsx [hunk 2]
```diff
@@ -758,7 +758,7 @@ class CurrentMetrics extends React.Component {
                            value={top_cores[0][1]}
                            className="current-top-cpu-usage pf-m-sm"
                            min={0} max={100}
-                           variant={ top_cores[0][1] > 90 ? ProgressVariant.danger : ProgressVariant.info }
+                           variant={ top_cores[0][1] > 90 ? ProgressVariant.danger : top_cores[0][1] >= 80 ? ProgressVariant.warning : ProgressVariant.info }
```
Comment (cockpituous) [inline pkg/metrics/metrics.jsx:761]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/metrics/metrics.jsx [hunk 3]
```diff
@@ -887,7 +887,7 @@ class CurrentMetrics extends React.Component {
                                     value={memTotal ? this.state.memUsed : undefined}
                                     className="pf-m-sm"
                                     min={0} max={memTotal}
-                                    variant={memUsedFraction > 0.9 ? ProgressVariant.danger : null}
+                                    variant={memUsedFraction > 0.9 ? ProgressVariant.danger : memUsedFraction >= 0.8 ? ProgressVariant.warning : null}
```
Comment (cockpituous) [inline pkg/metrics/metrics.jsx:890]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/metrics/metrics.jsx [hunk 4]
```diff
@@ -935,7 +935,7 @@ class CurrentMetrics extends React.Component {
                                         data-disk-usage-target={info.target}
                                         value={info.use} min={0} max={100}
                                         className="pf-m-sm"
-                                        variant={info.use > 90 ? ProgressVariant.danger : null}
+                                        variant={info.use > 90 ? ProgressVariant.danger : info.use >= 80 ? ProgressVariant.warning : null}
```
Comment (cockpituous) [inline pkg/metrics/metrics.jsx:938]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19217-20230822-113626-06b76265-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by martinpitt — APPROVED** [review]
Thanks! Garrett already approved  that design in #19000.

**Pull request merged**